### DISCUSSION TITLE
When will v0.155.0 be released

### DISCUSSION THREAD

**Discussion by Git4Mark** [helmfile/helmfile#905] [Questions] [answered]
When will v0.155.0 be released

**Comment by yxxhero** [comment]
@xiaomudk

**Reply by yxxhero** [reply]
@Git4Mark you are Chinese? me too.

**Reply by Git4Mark** [reply]
yes, i am chinese, how do you know that

**Reply by yxxhero** [reply]
your GitHub home page. thanks for using helmfile.

**Reply by Git4Mark** [reply]
I should be grateful,helmfile is a good tool

**Reply by Git4Mark** [reply]
So can we communicate in Chinese,my english is pool

**Reply by yxxhero** [reply]
no. please use English. you need more practice.

**Reply by yxxhero** [reply, marked as answer]
@Git4Mark we can make friends on Wechat: yuan_xiongxiong_xiong

**Reply by Git4Mark** [reply]
i can not find you on Wechat, can you find me weixin4mark

**Comment by xiaomudk** [comment]
@Git4Mark I am currently unable to determine the exact time, but I expect to complete it within a week.
